### PR TITLE
[cxx] Fix C++ build -- int vs. enum.

### PR DESCRIPTION
**Pull request by jaykrell**

### CODE REVIEW

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by lambdageek** [comment]
What do you think about adding a getter:
```c
 static MonoTypeEnum inst_c1_type (MonoInst *ins) { return (MonoTypeEnum)ins->inst_c1; }
```
and updating the 3 call sites instead?

**Comment by jaykrell** [comment]
Oh huh maybe messed up will look...

**Comment by jaykrell** [comment]
@monojenkins build failed

File: mono/mini/mini-llvm.c [hunk 1]
```diff
@@ -7588,7 +7594,7 @@ process_bb (EmitContext *ctx, MonoBasicBlock *bb)
 		}
 
 		case OP_SSE_LOADU: {
-			LLVMValueRef dst_ptr = convert (ctx, lhs, LLVMPointerType (primitive_type_to_llvm_type (ins->inst_c1), 0));
+			LLVMValueRef dst_ptr = convert (ctx, lhs, LLVMPointerType (primitive_type_to_llvm_type (inst_c1_type ins)), 0));
```
Comment (lambdageek) [inline mono/mini/mini-llvm.c]:
```suggestion
			LLVMValueRef dst_ptr = convert (ctx, lhs, LLVMPointerType (primitive_type_to_llvm_type (inst_c1_type (ins)), 0));
```

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins squash

**Comment by jaykrell** [comment]
@monojenkins build Windows x64 C++

**Comment by jaykrell** [comment]
@monojenkins squash

**Pull request merged**